readme.md

```diff
@@ -0,0 +1,105 @@
+# Intro
+
+This is MVP for one of our client (UTSEUS) which helps teams on hackathons exchange skills among team members and pay for their help.  It has three modules: web client (React), server (NodeJS) and storage with some business logic on blockchain (ethereum). This doc descriebs how to setup environment and run server backed by blockchian. 
+
+## Install environment
+
+A whole environment is described in scripts of package.json. Almost all what you need is run this scripts from the server folder. 
+
+```
+npm run install
+npm run testrpc
+npm run truffle
+# run server API tests
+npm run test-mocha
```

Comment:
fails:

```
rngadam@rngadam-mac ~/c/s/s/server> npm run test-mocha

> utseus_api@1.0.0 test-mocha /Users/rngadam/coderbunker/src/skills-market/server
> mocha --reporter spec

/Users/rngadam/coderbunker/src/skills-market/server/test/parser.test.js:6
test('test parser - get data', () => {
^

ReferenceError: test is not defined
    at Object.<anonymous> (/Users/rngadam/coderbunker/src/skills-market/server/test/parser.test.js:6:1)
    at Module._compile (module.js:573:32)
    at Object.Module._extensions..js (module.js:582:10)
    at Module.load (module.js:490:32)
    at tryModuleLoad (module.js:449:12)
    at Function.Module._load (module.js:441:3)
    at Module.require (module.js:500:17)
    at require (internal/module.js:20:19)
    at /Users/rngadam/coderbunker/src/skills-market/server/node_modules/mocha/lib/mocha.js:231:27
    at Array.forEach (native)
    at Mocha.loadFiles (/Users/rngadam/coderbunker/src/skills-market/server/node_modules/mocha/lib/mocha.js:228:14)
    at Mocha.run (/Users/rngadam/coderbunker/src/skills-market/server/node_modules/mocha/lib/mocha.js:536:10)
    at Object.<anonymous> (/Users/rngadam/coderbunker/src/skills-market/server/node_modules/mocha/bin/_mocha:573:18)
    at Module._compile (module.js:573:32)
    at Object.Module._extensions..js (module.js:582:10)
    at Module.load (module.js:490:32)
    at tryModuleLoad (module.js:449:12)
    at Function.Module._load (module.js:441:3)
    at Module.runMain (module.js:607:10)
    at run (bootstrap_node.js:420:7)
    at startup (bootstrap_node.js:139:9)
    at bootstrap_node.js:535:3
```

Reply:
Done.Remove mocha tests from consideration, jest pick up it on the flight